test/scripts/start-services.sh

```diff
@@ -61,40 +64,51 @@ deploy_contracts()
 
 start_parachain()
 {
-    echo "Starting Parachain"
+    echo "Build Relaychain"
```

Comment:
I'd prefer that this be done in a higher-level "bootstrap" script that does the following:
1. Installs polkadot-launch
2. Installs & builds polkadot
3. Runs the start-services script

Reply:
yes make sense.  notice that the polkadot binary is a forked release and seems currently polkadot enable beefy by default Have you guys made some other customization or can we just use the official release?

```
git clone https://github.com/snowfork/polkadot.git /tmp/polkadot
pushd /tmp/polkadot
git checkout enable_beefy_on_rococo
cargo build --release
```